src/Tools/dotnet-counters/README.md

```diff
@@ -0,0 +1,102 @@
+# dotnet-counters
+
+
+## Install dotnet-counters
+
+```
+ dotnet tool install -g dotnet-counters
+```
+
+
+## Intro
+
+dotnet-counters is a performance monitoring tool for ad-hoc health monitoring or 1st level performance investigation. It can observe performance counter values that are published via `EventCounter` API (https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventcounter). 
+
```

Comment:
Lets add a few common example counters to the text to give a feel for what is possible.